rfd/0109-cloud-agent-upgrades.md

```diff
@@ -327,6 +327,30 @@ to be changed to follow the same pattern of registering a release channel and in
 by ec2 discovery.
 
 
+### ECS Fargate Agents
+
+When a user uses the Teleport Discover flow to connect an RDS, a Teleport agent is automatically provisioned using ECS Fargate. We need a
+way to keep these agents updated. One idea is to run a separate goroutine in Teleport that periodically checks for an available update and
```

Comment:
`goroutine` seems like a too low level implementation detail, maybe we can just call it `separate periodical check`?